ui/src/shared/components/DatabaseList.tsx

```diff
@@ -52,18 +52,13 @@ class DatabaseList extends Component<DatabaseListProps, DatabaseListState> {
 
   public componentDidUpdate({
     querySource: prevSource,
-    query: prevQuery,
   }: {
     querySource?: Source
     query: QueryConfig
   }) {
     const {querySource: nextSource, query: nextQuery} = this.props
     const differentSource = !_.isEqual(prevSource, nextSource)
 
-    if (prevQuery.rawText === nextQuery.rawText) {
-      return
-    }
```

Comment:
how come this was removed?

Reply:
i actually do want the query to run again if I change sources and the query is still the same.